.NET Core/Embed for your organization/UserOwnsData/appsettings.json

```diff
@@ -1,15 +1,21 @@
 {
   "AzureAd": {
-    "Instance":  "",
-    "AuthorityUrl": "https://login.microsoftonline.com/",
+    "Instance":  "https://login.microsoftonline.com/",
     "Domain": "",
     "TenantId": "common",
-    "Scopes": ["https://analysis.windows.net/powerbi/api/Report.Read.All https://analysis.windows.net/powerbi/api/Dashboard.Read.All https://analysis.windows.net/powerbi/api/Workspace.Read.All"],
+    "Scopes": "https://analysis.windows.net/powerbi/api/.default",
     "ClientId": "",
     "ClientSecret": "",
-    "CallbackPath": "/signin-oidc"
+    "CallbackPath": "/signin-oidc",
+    "SignedOutCallbackPath": "/signout-callback-oidc"
+  },
+  "PowerBi": {
+    "ServiceRootUrl": "https://api.powerbi.com/"
+  },
+  "DownstreamApi": {
```

Comment:
Not sure why you rename this as DownstreamApi ? Any specific reason ? I prefer to name it as MicrosoftGraphAPI.


Remove this. We can move it to the startup.cs file. Check the comment there.